pom.xml

```diff
@@ -2656,7 +2656,8 @@
       <id>kubernetes</id>
       <modules>
         <module>resource-managers/kubernetes/core</module>
-        <module>resource-managers/kubernetes/integration-tests</module>
+        <!-- for now, don't automatically build and run the integration tests -->
+        <!-- <module>resource-managers/kubernetes/integration-tests</module> -->
```

Comment:
I think this would mean it's not possible to run the integration tests at all via Maven, because the integration tests module will never be available. What about creating a separate profile, `kubernetes-integration-tests`, which includes both the `kubernetes` and the `kubernetes/integration-tests` module?